src/components/Button/Button.tsx

```diff
@@ -27,6 +27,7 @@ export type ButtonPropsT = {
   active?: boolean;
   id?: string;
   text?: string;
+  label?: string;
```

Comment:
I think I'd make accessibility fields required in the type.

Reply:
yea I figured you'd ask this hah, I tend to like defaults but yea I can change that.